specification/maps/data-plane/Weather/stable/1.1/examples/GetActiveStorms.json

```diff
@@ -1,6 +1,6 @@
 {
   "parameters": {
-    "api-version": "2.0",
+    "api-version": "1.1",
```

Comment:
I thought the discussion at the meeting was for date-based versions since that's what other services with an API version query parameter do?

Reply:
Per the discussion and conclusion in the 2nd weather api review session, we would apply api version 1.1 and I've shared the related recording with Travis.